src/app/ui-components/sidebar-right/sidebar-right.component.ts

```diff
@@ -87,4 +87,8 @@ export class SideBarRightComponent implements OnInit, OnDestroy {
             },
         );
     }
+
+    toLowerCase(label: string): string {
```

Comment:
turn this into a pipe

Reply:
remove this function and call toLowerCase function on the label and consider using a pipe to turn spaces into dashes